src/components/CreateFamily/FamilyForm/CreateFamilyModal.cy.tsx

```diff
@@ -0,0 +1,63 @@
+import { noop } from '@fullstacksjs/toolbox';
+
+import { CreateFamilyModal } from './CreateFamilyModal';
+
+const requiredFieldMessage = 'این فیلد ضروری است';
+const minLengthMessage = 'نام خانواده باید حداقل ۵ حرف باشد';
+const buttonSelector = '[data-test="submit-button"]';
+const inputSelector = '[data-test="family-name"]';
+
+describe('CreateFamilyModal', () => {
+  beforeEach(() => {
+    cy.mount(<CreateFamilyModal opened onClose={noop} />);
+  });
+  it('should contains text input to enter family name', () => {
+    cy.get('form').get(inputSelector).should('match', 'input');
+  });
+  it('should contains submit button to create family', () => {
+    cy.get('form')
+      .get(buttonSelector)
+      .should('have.attr', 'type')
+      .and('equal', 'submit');
+  });
+  it('should not show a required error message when family name is not empty', () => {
+    cy.get('form').within(() => {
+      cy.get(inputSelector);
+      cy.root().submit();
+      cy.findByText(`/${requiredFieldMessage}/`).should('not.exist');
+    });
+  });
+  it('should show a required error message when family name is empty', () => {
+    cy.get('form').within(() => {
+      cy.root().submit();
+      cy.findByRole('alert').should('have.text', requiredFieldMessage);
+    });
+  });
+  it('should show an error message when family name is less than min length', () => {
+    cy.get('form').within(() => {
+      cy.get(inputSelector).type('م');
+      cy.findByRole('alert').should('have.text', minLengthMessage);
+    });
+  });
+
+  it('should not show an error message when family name is more than or equal min length', () => {
+    cy.get('form').within(() => {
+      cy.get(inputSelector).type('مرادی');
+      cy.findByText(`/${minLengthMessage}/`).should('not.exist');
+    });
+  });
+
+  it('should be disabled the button when the required fields are filled in incorrectly', () => {
```

Comment:
when you use `it` function, the function name refers to what you used in `describe` function at the top
so it would be like this:

CreateFamilyModal should be disabled the button when the required fields are filled in incorrectly

I think you should change your title

Reply:
Hmmm. that's good point. 
Is this a better title? `should disable the button when the required fields are filled incorrectly`

Reply:
We have two scenarios in our Form when button should be disable:
First when user type less then 3.
Second when user submit the form without filling the input

So This test, I mean `should disable the button when the required fields are filled incorrectly` is duplication and I have to remove it. u agree?